docs/dataprocessing/intro.rst

```diff
@@ -24,7 +24,7 @@ Structure
 
 The **dataprocessing** module uses a base class (**DataProcessing**) that implements several tasks shared
 across the different transformation classes. Whenever possible, the classes implemented in this module follow
-the .fit() and .transform() behavior from scikit-learn, making these classes compatible with scikit-learn.
+the ``.fit()`` and ``.transform()`` behavior from scikit-learn, making these classes compatible with scikit-learn.
```

Comment:
I think it makes sense to put ``scikit-learn``, just to show that this is a package.

Reply:
I ended up not making this change since I noted that other libraries were also written using a normal font. This way, if I change this one, I'll need to change the font of all libraries, and I might end up missing one or two library names. So, I decided to leave it as is to avoid these inconsistencies.